src/views/partials/board-settings.handlebars

```diff
@@ -0,0 +1,42 @@
+<a href="/board/{{board._id}}">Back</a>
+<div id="boardSettings">
+    <!-- Form for changing the board settings -->
+    <form action="/board/settings/{{board._id}}" method="POST">
```

Comment:
this is a PATCH request in the routes, but we can change it there, doesn't really matter to me. Or else use the hidden input element to change this form to a PATCH request

Reply:
This is not needed. You will need to do an ajax request anyway and you can use PATCH through that.